akka-http2-support/src/main/scala/akka/http/impl/engine/http2/framing/Http2FrameParsing.scala

```diff
@@ -7,21 +7,43 @@ package framing
 
 import scala.collection.immutable
 
-import akka.stream.Attributes
```

Comment:
Some place in our code will have to generate the PUSH_PROMISE frame so we would handle it there.